src/app/common/PolicyTypeTag.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <span class="policy-type-tag">
+    <img
+      v-if="policy.iconUrl !== null"
+      class="policy-type-tag-icon"
+      :src="policy.iconUrl"
+      alt=""
+    >
+
+    <KIcon
+      v-else
+      icon="brain"
+      size="24"
+    />
```

Comment:
IIUC the "brain" icon is shown whenever there's no dedicated icon for this policy?

Reply:
Yes, we’re missing 5 icons in total (the new Mesh* policies, basically). I don’t know who creates them for kuma.io, but we probably can get some new ones, right?

Reply:
Yeah we could. I think for the new policies we can reuse the old ones (as they are replacements really). But in any case we should keep the old stuff just because there's usually a delay between icons and actual implem (and we're also disucssing having this fully pluggable for users)

Reply:
I’ve updated the following to use existing icons of their predecessors:

- MeshAccessLog → TrafficLog
- MeshTrace → TrafficTrace
- MeshTrafficPermission → TrafficPermission